Project/App/Sources/Feature/Onboarding/SignUp/BirthTime/BirthTimeReducer.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  BirthTimeReducer.swift
+//  ProductName
+//
+//  Created by hyerin on 6/13/25.
+//
+
+import Foundation
+
+import ComposableArchitecture
+
+@Reducer
+struct BirthTimeReducer {
+  init() {}
+
+  @ObservableState
+  struct State: Equatable {
+    var birthTime: Date
+    var isNoIdeaButtonChecked: Bool
```

Comment:
```suggestion
    var hasNoIdeaButtonChecked: Bool
```
is -> has가 문법상 더 맞는 것 같아요